papers/Phi Fei Fo Fum (WIP)/paper.Rmd

```diff
@@ -0,0 +1,258 @@
+
+$$
+\chi^2 = \sum_{i=1}^{l\times k}{\frac{(O_i-E_i)^2}{E_i}}
+$$
+
+Where $O_i$ are the *observed* frequencies and $E_i$ are the frequencies *expected* under independence, and $l$ and $k$ are the number of rows and columns of the contingency table. 
+
+Instead of the deviations between the observed and expected frequencies, we can write $\chi^2$ in terms of observed and expected cell *probabilities* and the total sample size $N$ (since $p=k/N$):
+
+$$
+\chi^2 = N\times\sum_{i=1}^{l\times k}{\frac{(p_{O_i}-p_{E_i})^2}{p_{E_i}}}
+$$
+
+Where $p_{O_i}$ are the *observed* cell probabilities and $p_{E_i}$ are the probabilities *expected* under independence.
+
+For example, which might ask of the probability of survival of the sinking of the Titanic is dependant on the sex of the passenger:
+
+```{r}
+(Titanic_xtab <- as.table(apply(Titanic, c(2, 4), sum)))
+
+chisq.test(Titanic_xtab)
+```
+
+## Phi
+
+For a 2-by-2 contingency table analysis, like the one used above, the $\phi$ (*phi*) coefficient is a correlation-like measure of effect size indicating the strength of association between the two binary variables. One way to compute this effect size is to re-code the binary variables as dummy (0, 1) variables, and computing the (absolute) Pearson correlation between them:
+
+$$
+\phi = |r_{AB}|
+$$
+
+Another way to compute $\phi$ is by using the $\chi^2$ statistic:
+
+$$
+\phi = \sqrt{\frac{\chi^2}{N}} = \sqrt{\sum_{i=1}^{l\times k}{\frac{(p_{O_i}-p_{E_i})^2}{p_{E_i}}}}
+$$
+
+This value ranges between 0 (no association) and 1 (complete dependence), and its values can be interpreted the same as Person's correlation coefficient.
+
+```{r}
+library(effectsize)
+library(correlation)
+
+phi(Titanic_xtab, adjust = FALSE)
+
+tidyr::uncount(as.data.frame(Titanic_xtab), weights = Freq) |>
+  transform(Survived = Survived == "Yes",
+            Sex = Sex == "Male") |> 
+  correlation()
+```
+
+## Cramér's *V*
+
+These properties do not hold when the contingency table is larger than 2-by-2: $\sqrt{\chi^2/N}$ can be larger than 1.
+Cramér showed [@cramer1999mathematical] that while for 2-by-2 the maximal possible value of $\chi^2$ is $N$, for larger tables the maximal possible value for $\chi^2$ is $N\times (\text{min}(k,l)-1)$.
+Therefore, he suggested the $V$ effect size (also sometimes known as Cramér's phi and denoted as $\phi_{c}$):
+
+$$
+\text{Cramer's } V = \sqrt{\frac{\chi^2}{N(\text{min}(k,l)-1)}}
+$$
+
+$V$ is 1 when the columns are completely dependent on the rows, or the row are completely dependent on the columns.
+
+```{r}
+(Titanic_xtab2 <- as.table(apply(Titanic, c(1, 4), sum)))
+
+cramers_v(Titanic_xtab2, adjust = FALSE)
+```
+
+Tschuprow [@tschuprow1939principles] devised an alternative value, at <value><!-- doesn't render -->, which is 1 only when the columns are completely dependent on the rows *and* the rows are completely dependent on the columns, which is only possible when the contingency table is a square.
```

Comment:
We should give a counter example of V and Tschuprow tables where V is 1 but the other is not.